src/paint.ts

```diff
@@ -30,3 +30,13 @@ export const initLargestContentfulPaint = (
     lcp.value = lastEntry.renderTime || lastEntry.loadTime;
   }
 };
+
+export const initElementTiming = (
+  performanceEntries: IPerformanceEntry[],
+) => {
+  performanceEntries.forEach(performanceEntry => {
+    if (performanceEntry.identifier) {
```

Comment:
To keep Perfume.js super tiny I have Terser at max modification, so to make sure this code working as **minified** as well, we need to add the word `identifier` into this list https://github.com/Zizzamia/perfume.js/blob/master/rollup.config.ts#L80.

Otherwise, we will get `identifier` minified to a different single letter. 😅

Reply:
It's weird I can see the `.identifer` in the minified output _before_ the change

![Screenshot_053120_114742_AM](https://user-images.githubusercontent.com/3313870/83350546-b6e04780-a334-11ea-9452-4492531f0c57.jpg)

Based on what you said and the terser doc's I'd expect that to be minified, no worries I've added it anyway: [https://github.com/Zizzamia/perfume.js/pull/124/commits/f10f58642ba63af04e7a34990c3bd4533f68f98b ](https://github.com/Zizzamia/perfume.js/pull/124/commits/f10f58642ba63af04e7a34990c3bd4533f68f98b)

Reply:
Good research!  For the next release, it just safer have it. But in a future separate PR we should probably trim out all the ones that are not really necessary anymore. Thank you.